admin-app/src/main/resources/application.conf

```diff
@@ -68,11 +68,12 @@ drill {
             buildTestingReportPath = "/dashboard/2"
             buildTestingReportPath = ${?DRILL_METRICS_UI_BUILD_TESTING_REPORT_PATH }
         }
-        scheduler {
-            refreshViewsIntervalInMinutes = 30
-            refreshViewsIntervalInMinutes = ${?DRILL_METRICS_SCHEDULER_REFRESH_VIEWS_INTERVAL_IN_MINUTES}
-        }
     }
     testRecommendations {
     }
+    scheduler {
+        refreshViewsIntervalInMinutes = 30
+        refreshViewsIntervalInMinutes = ${?DRILL_METRICS_SCHEDULER_REFRESH_VIEWS_INTERVAL_IN_MINUTES} //todo DRILL_METRICS_SCHEDULER_REFRESH_VIEWS_INTERVAL_IN_MINUTES is deprecated, use DRILL_SCHEDULER_REFRESH_VIEWS_INTERVAL_IN_MINUTES
```

Comment:
I moved the `scheduler` property according to the current location of the scheduler code. But I decided to keep the old env variable for backward compatibility since we have clients

Reply:
we dont have anyone using this yet, so we can change that